src/lsrs/web/controller/actuator-controller.ts

```diff
@@ -25,7 +25,7 @@ export default class ActuatorController implements Controller {
      *
      * @returns application info as InfoResponse wrapped in ResponseWrapper
      */
-    public info(): ResponseWrapper<InfoResponse> {
+    public async info(): Promise<ResponseWrapper<InfoResponse>> {
```

Comment:
Why is this async?

Reply:
Your suggestion regarding the MappingHelper#register methods made it possible to revert this change, so thanks! :)